doc8/parser.py

```diff
@@ -80,17 +80,20 @@ def _read(self):
         with self._read_lock:
             if not self._has_read:
                 with open(self.filename, "rb") as fh:
-                    self._lines = list(fh)
-                    fh.seek(0)
                     self._raw_content = fh.read()
+                    self._lines = self._raw_content.splitlines(True)
```

Comment:
++

Reply:
What did "++" mean here?

Reply:
Just that it was a good improvement

Reply:
Thanks, I'm familiar with +1 in that context, this was new to me.